metricbeat/module/vsphere/datastorecluster/_meta/data.json

```diff
@@ -0,0 +1,27 @@
+{
+    "@timestamp": "2016-05-23T08:05:34.853Z",
+    "service": {
+        "address": "https://localhost:8980/sdk",
+        "type": "vsphere"
+    },
+    "event": {
+        "dataset": "vsphere.datastore_cluster",
+        "module": "vsphere",
+        "duration": 15443161
+    },
+    "metricset": {
+        "period": 20000,
+        "name": "datastorecluster"
+    },
+    "vsphere": {
+        "datastorecluster": {
+            "name": "datastore_cluster1",
+            "capacity": {
```

Comment:
Shouldn't those also be `datastorecluster`?

Reply:
Yes, I missed it. i have updated it now.